charts/jaeger-operator/templates/service.yaml

```diff
@@ -14,4 +14,7 @@ spec:
   selector:
     app.kubernetes.io/name: {{ include "jaeger-operator.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
-  type: ClusterIP
+  type: {{ .Values.service.type }}
+    {{- if and (eq .Values.service.type "NodePort") (.Values.service.nodePort) -}}
+    nodePort: {{ .Values.service.nodePort }}
```

Comment:
Shouldn't this be moved to the port description?

```
ports:
  - name: metrics
    port: 8383
    protocol: TCP
    targetPort: 8383
    {{ if (and (eq .Values.service.type "NodePort") (not (empty .Values.service.nodePort))) }}
    nodePort: {{.Values.service.nodePort}}
    {{- end }}
```

Reply:
Done 👍